.gitignore

```
@@ -170,3 +169,5 @@ fabric.properties
 !/gradle/wrapper/gradle-wrapper.jar
 
 # End of https://www.toptal.com/developers/gitignore/api/androidstudio,kotlin
+
+python/data/
```

We can avoid having to remove `*.zip` on line 38, by overriding the rule in the specific directory, like so:
```suggestion
!python/**
python/data
```
`python/data` is still there, so that specific directory is ignored.